gradle.properties

```diff
@@ -21,5 +21,4 @@ org.gradle.jvmargs=-Xmx8g
 # silence the "multiplatform alpha" warning
 kotlin.mpp.stability.nowarn=true
 
-# https://github.com/gradle/gradle/issues/11412
-systemProp.org.gradle.internal.publish.checksums.insecure=true
```

Comment:
Another workaround bites the dust. This shouldn't be needed anymore now that we're publishing to MavenCentral only